src/components/Organization/Dashboard/Layout.tsx

```diff
@@ -49,7 +49,19 @@ const OrganizationDashboardLayout: React.FC = () => {
                       <Image src={fallback} />
                     </AspectRatio>
                   )}
-                  <OrganizationName fontSize='text' fontWeight={600} m={0} color='dashboard.org_name' />
+                  <OrganizationName
+                    fontSize='text'
+                    fontWeight={600}
+                    m={0}
+                    color='dashboard.org_name'
+                    sx={{
+                      display: '-webkit-box',
+                      WebkitLineClamp: 2,
+                      WebkitBoxOrient: 'vertical',
```

Comment:
You should not be using webkit hacks, despite the fact they work on most browsers (in some browsers won't work).